app/src/main/java/fr/jaetan/jbudget/app/home/FabItem.kt

```diff
@@ -3,4 +3,4 @@ package fr.jaetan.jbudget.app.home
 import androidx.annotation.StringRes
 import androidx.compose.ui.graphics.vector.ImageVector
 
-class FabItem(@StringRes val text: Int, @StringRes val descriptor: Int, onClick: () -> Unit, icon: ImageVector)
\ No newline at end of file
+class FabItem(@StringRes val text: Int, @StringRes val descriptor: Int,val onClick: () -> Unit,val icon: ImageVector)
```

Comment:
Rajoute un espace entre val et la virgule
Et, on reverra ensemble le visuel sur FAB quand il est déroulé

Reply:
Noté, tu veux que je push le commit de correction directement ou pas besoins ?

Reply:
pas besoin tqt